stream-chat-android/src/main/java/com/getstream/sdk/chat/viewmodel/factory/ChannelsViewModelFactory.kt

```diff
@@ -27,7 +27,7 @@ public class ChannelsViewModelFactory(
     private val limit: Int = 30
 ) : ViewModelProvider.Factory {
     override fun <T : ViewModel?> create(modelClass: Class<T>): T {
-        require(modelClass == ChannelsViewModel::class) {
```

Comment:
Really feels like this should've promoted an IDE warning about mismatching types in a comparison...